ddtrace/contrib/boto/__init__.py

```diff
@@ -24,22 +24,6 @@
     True`` or by setting the environment variable ``DD_AWS_TAG_NO_PARAMS=true``.
 
 
-    Default: ``False``
```

Comment:
?

Reply:
This is a weird github diff, line 27 (previously line 43) is still remaining.